package.json

```diff
@@ -27,6 +27,7 @@
     "react": "^18.2.0",
     "react-copy-to-clipboard": "^5.1.0",
     "react-dom": "^18.2.0",
+    "react-linkify": "^1.0.0-alpha",
```

Comment:
@buckyroberts I tried writing my own function to linkify text with regex, but then encountered a few use-cases in which it wasn't working really well.
So used this light-weight package instead.

Is this fine?